src/main.rs

```diff
@@ -52,6 +52,7 @@ fn version() -> anyhow::Result<()> {
 }
 
 async fn proxy(cfg: config::Config) -> anyhow::Result<()> {
+    info!("version: {}", version::BuildInfo::new());
```

Comment:
there is a function `version` can be called

Reply:
It's useful to have it in the logs so when someone posts a log in an issue we know what version was ran. We have this for all other Istio components, and it prevented me from debugging an issue today because it was missing

Reply:
that does print, not log